nova/core/roles/machine_operations/defaults/main.yml

```diff
@@ -68,7 +68,7 @@ vmwarews_custom_params: []
 ################
 
 # Temporary ssh key path on your container
-temp_ssh_key_path: /tmp/{{ project_fullname }}_{{ inventory_hostname }}_aws_key
+temp_ssh_key_path: /tmp/{{ project_fullname | default('') }}_{{ inventory_hostname }}_aws_key
```

Comment:
If `project_fullname` is not defined, then file will start with `_`

Reply:
yes that's is fine